library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -97,13 +97,23 @@
   private List<CalendarCellDecorator> decorators;
   private DayViewAdapter dayViewAdapter = new DefaultDayViewAdapter();
 
+  private boolean monthsReverseOrder;
+
   public void setDecorators(List<CalendarCellDecorator> decorators) {
     this.decorators = decorators;
     if (null != adapter) {
       adapter.notifyDataSetChanged();
     }
   }
 
+  public boolean isMonthsReverseOrder() {
+    return monthsReverseOrder;
+  }
+
+  public void setMonthsReverseOrder(boolean monthsReverseOrder) {
+    this.monthsReverseOrder = monthsReverseOrder;
```

Comment:
you'll either need to add
```
    if (null != adapter) {
      adapter.notifyDataSetChanged();
    }
```
to this method, or just remove the method.  I'm not sure it'll ever really be necessary to change this on the fly, so you probably can just remove this (and for that matter, `isMonthsReverseOrder()` too)